components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2122,13 +2126,26 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
             },
         );
         if (isCreateStripeCustomerOnUsageEnabled) {
+            let customer: StripeCustomer | undefined;
```

Comment:
Currently we'd handle this only for the feature flag enabled branch. For compatibility, we should either also handle it on the non enabled branch, or wait until next week when we're planning to remove this feature flag and default to the billing service.

Reply:
> or wait until next week

let's do that.

Reply:
Removal is [here](https://github.com/gitpod-io/gitpod/pull/14060)